src/experimental/constructs/error-budget-alarm.test.ts

```diff
@@ -16,6 +16,32 @@ describe("The ErrorBudgetAlarmExperimental construct", () => {
     expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
   });
 
+  it("should fail to create Alarm when slo target is above 99.95%", () => {
+    const stack = simpleGuStackForTesting();
+    expect(() => {
+      new GuErrorBudgetAlarmExperimental(stack, {
+        sloName: "MapiFrontsAvailability",
+        sloTarget: 0.9996,
+        badEvents: new Metric({ metricName: "HttpErrors", namespace: "TestLoadBalancerMetrics" }),
+        validEvents: new Metric({ metricName: "HttpRequests", namespace: "TestLoadBalancerMetrics" }),
+        snsTopicNameForAlerts: "test-sns-topic",
+      });
+    }).toThrowError();
```

Comment:
Can we add the error message that's thrown to the test, to be sure only the expected error is seen?